capa/ida/plugin/form.py

```diff
@@ -285,10 +321,16 @@ def load_view_status_label(self):
 
         self.view_status_label = label
 
+    def load_result_cache_label(self):
+        """load result store label"""
+        label = QtWidgets.QLabel()
+        label.setAlignment(QtCore.Qt.AlignLeft)
+        self.view_result_cache_label = label
+
     def load_view_buttons(self):
         """load the button controls"""
         analyze_button = QtWidgets.QPushButton("Analyze")
-        reset_button = QtWidgets.QPushButton("Reset")
+        reset_button = QtWidgets.QPushButton("Reset selections/highlights")
```

Comment:
We overload the reset button for both tabs and it completes different tasks depending on the selected tab. This is another instance calling for us to rework the buttons so it's less confusing for the user.